android/sdl_android/src/androidTest/java/com/smartdevicelink/managers/screen/AlertManagerTest.java

```diff
@@ -0,0 +1,170 @@
+public class AlertManagerTest {
+    AlertManager alertManager;
+
+    @Before
+    public void setUp() throws Exception {
+        Context mTestContext = getInstrumentation().getContext();
+
+        // mock things
+        ISdl internalInterface = mock(ISdl.class);
+        FileManager fileManager = mock(FileManager.class);
+        PermissionManager permissionManager = mock(PermissionManager.class);
+
+        when(internalInterface.getPermissionManager()).thenReturn(permissionManager);
+
+        Taskmaster taskmaster = new Taskmaster.Builder().build();
+        taskmaster.start();
+        when(internalInterface.getTaskmaster()).thenReturn(taskmaster);
+
+        Answer<Void> permissionAnswer = new Answer<Void>() {
+            @Override
+            public Void answer(InvocationOnMock invocation) {
+                Object[] args = invocation.getArguments();
+                OnPermissionChangeListener onPermissionChangeListener = (OnPermissionChangeListener) args[2];
+                Map<FunctionID, PermissionStatus > allowedPermissions = new HashMap<>();
+                int permissionGroupStatus = PermissionManager.PERMISSION_GROUP_STATUS_DISALLOWED;
+                onPermissionChangeListener.onPermissionsChange(allowedPermissions,permissionGroupStatus);
+                return null;
+            }
+        };
+        doAnswer(permissionAnswer).when(permissionManager).addListener(any(List.class), anyInt(), any(OnPermissionChangeListener.class));
+
+        Answer<Void> onSystemCapabilityAnswer = new Answer<Void>() {
+            @Override
+            public Void answer(InvocationOnMock invocation) {
+                Object[] args = invocation.getArguments();
+                OnSystemCapabilityListener onSystemCapabilityListener = (OnSystemCapabilityListener) args[1];
+                WindowCapability windowCapability = getWindowCapability(3);
+                DisplayCapability displayCapability = new DisplayCapability();
+                displayCapability.setWindowCapabilities(Collections.singletonList(windowCapability));
+                List<DisplayCapability> capabilities = Collections.singletonList(displayCapability);
+                onSystemCapabilityListener.onCapabilityRetrieved(capabilities);
+                return null;
+            }
+        };
+
+        SystemCapabilityManager systemCapabilityManager = mock(SystemCapabilityManager.class);
+        doAnswer(onSystemCapabilityAnswer).when(systemCapabilityManager).addOnSystemCapabilityListener(eq(SystemCapabilityType.DISPLAYS), any(OnSystemCapabilityListener.class));
+        doReturn(systemCapabilityManager).when(internalInterface).getSystemCapabilityManager();
+
+        alertManager = new AlertManager(internalInterface, fileManager);
+    }
+
+    @Test
+    public void testInstantiation() {
+        assertNotNull(alertManager.defaultMainWindowCapability);
+        assertNotNull(alertManager.nextCancelId);
+        assertFalse(alertManager.currentAlertPermissionStatus);
```

Comment:
```suggestion
        assertFalse(alertManager.isAlertRPCAllowed);
```
Missed during a refactor